### PR TITLE
Update the desktop CRA template with local briefcase support

### PR DESCRIPTION
**Pull request by kckst8**
Adds support for local briefcases (the same code from the previously reviewed test app)

### CODE REVIEW

File: packages/modules/cra-template-desktop-viewer/template.json [hunk 1]
```diff
@@ -34,10 +34,10 @@
       "@bentley/ui-framework": "^2.14.0",
       "@bentley/ui-ninezone": "^2.14.0",
       "@bentley/webgl-compatibility": "^2.14.0",
-      "@itwin/desktop-viewer-react": "^1.0.16",
+      "@itwin/desktop-viewer-react": "^1.2.0",
```
Comment (aruniverse) [inline packages/modules/cra-template-desktop-viewer/template.json:37]: i think moving forward (after 3.0) we can just use the `latest` dist tag here

File: packages/modules/cra-template-desktop-viewer/template/src/react-app-env.d.ts [hunk 2]
```diff
@@ -0,0 +1,6 @@
+/*---------------------------------------------------------------------------------------------
```
Comment (aruniverse) [inline packages/modules/cra-template-desktop-viewer/template/src/react-app-env.d.ts:1]: i dont think you need this file, should get auto generated when you create a new app using our cra template and fork

**Review by aruniverse — APPROVED** [review]

File: packages/modules/cra-template-desktop-viewer/template.json (hunk 1 above)
Comment (kckst8) [reply packages/modules/cra-template-desktop-viewer/template.json:37]: agree, as long as we're still using the next tag for prerelease

File: packages/modules/cra-template-desktop-viewer/template/src/react-app-env.d.ts (hunk 2 above)
Comment (kckst8) [reply packages/modules/cra-template-desktop-viewer/template/src/react-app-env.d.ts]: Mistake copy. I deleted

**Pull request merged**